app/views/partials/liabilities/activity_tab_events_table.scala.html

```diff
@@ -60,6 +60,15 @@ <h3 class="heading-medium mt-30 mb-30">@messages("case.v2.liability.activity_tab
                         case EventType.CASE_COMPLETED => {
                             activity_case_completed(index, event)
                         }
+
+                        case EventType.CASE_CREATED => {
+                        val body = event.details.asInstanceOf[CaseCreated]
+
+                        <div id="activity-events-row-case_created">
+                            <span id="activity-events-row-application_submitted-comment" class="pre-wrap">{body.comment}</span>
+                        </div>
```

Comment:
Same question here

Reply:
Good point, it now uses progressive_disclosure 👍